utils/enrich.py

```diff
@@ -75,13 +75,20 @@ def go_doc(fp):
 @click.command()
 @click.argument('input_files', type=click.Path(exists=True), nargs=-1)
 @click.option('--es', 'elasticsearch', default=False, is_flag=True, help="Include elasticsearch indexing command before each sentence.")
-def main(input_files, elasticsearch=False):
+@click.option('--mongo', 'mongo', default=False, is_flag=True, help="Include _id field in each sent.")
+def main(input_files, elasticsearch=False, mongo=False):
     for fp in input_files:
         sents = go_doc(fp)
         for sent in sents:
             if elasticsearch:
-                click.echo(to_json({'index': {'_id' : sent_id(sent[DOC_ID_KEY], sent[SENT_ID_KEY])}}))
-            click.echo(to_json(sent))
+                click.echo(
+                    to_json({'index': {'_id': sent_id(sent[DOC_ID_KEY], sent[SENT_ID_KEY])}}))
+                click.echo(to_json(sent))
+            elif mongo:
+                sent['_id'] = sent_id(sent[DOC_ID_KEY], sent[SENT_ID_KEY])
+                click.echo(to_json(sent))
+            else:
+                click.echo(to_json(sent))
```

Comment:
talvez usar https://click.palletsprojects.com/en/7.x/options/#choice-options? nada mt importante though

Reply:
done in https://github.com/own-pt/sensetion.el/pull/160/commits/df58e590a2b053b9827968d71e42c64dfd41136c, okay @hmuniz ?